### ISSUE TITLE
how many epochs do you train to get your final model?

### ISSUE THREAD

**Issue by stojakovic312636841** [thstkdgus35/EDSR-PyTorch#35]
hello,
         i have another question. 
         how many epochs do you train to get your final model(EDSRx2.pt/EDSRx3.pt/EDSRx4.pt)?

 which indicator determines that the training is completed? loss or PSNR?
and what the value of the loss and PSNR when the model has been trained completed?

thank you

**Comment by thstkdgus35** [comment]
Hello.

We trained our model for 650 epochs.

You can use either PSNR or loss to check whether the model converged or not.

Also, please check our paper to see the final performance.

Thank you.
